meinberlin/apps/votes/models.py

```diff
@@ -78,6 +80,11 @@ def is_valid(self, model):
     def is_valid_for_item(self, item):
         return item.module == self.module and self.is_valid(item.__class__)
 
+    def __str__(self):
+        return '{}-{}-{}'.format(self.token[0:4],
+                                 self.token[4:8],
+                                 self.token[8:13])
```

Comment:
This should only go up to 12, `self.token[8:12]` or to the end `self.token[8:]`

Reply:
Haha, right, good catch!